src/Yesod/Auth/OAuth2/ORCID.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Yesod.Auth.OAuth2.Orcid
```

Comment:
I see the module name is `Orcid` but the file is `ORCID.hs`. I think these should match. I worry about problems on MacOS (case-insensitive file system). I would probably use `ORCID` consistently.

Reply:
I've corrected the module name.